scripts/do-release.sh

```diff
@@ -110,7 +110,7 @@ function tag_images() {
     # Creating a local tag so that images are uploaded with it
     git tag -a -f "${release['version']}" -m "${release['version']}"
```

Comment:
```suggestion
    _git tag -a -f "${release['version']}" -m "${release['version']}"
```

Either this or just call the whole function with `in_project_repo`